app/forms/waste_carriers_engine/contact_phone_form.rb

```diff
@@ -4,9 +4,11 @@ module WasteCarriersEngine
   class ContactPhoneForm < BaseForm
     attr_accessor :phone_number
 
+    validates :phone_number, "defra_ruby/validators/phone_number": true
+
     def initialize(transient_registration)
       super
-      self.phone_number = @transient_registration.phone_number
+      self.phone_number = transient_registration.phone_number
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.phone_number = transient_registration.phone_number
```